### DISCUSSION TITLE
Can't deploy Medusa V2 to DigitalOcean App Platform. self-signed certificate in certificate chain.

### DISCUSSION THREAD

**Discussion by mrguruo** [medusajs/medusa#10282] [Q&A]
I am trying to deploy Medusa V2, pretty much as is out of the box to DigitalOcean's App Platform.
I have this script setup in the Run Command:
cd .medusa/server && npm run predeploy && npm run start
And I also added the redeploy script as per the General Deployment Docs.

The error I get is:
```
medusa-starter-default@0.0.1 predeploy
medusa db:migrate
{"level":"error","message":"self-signed certificate in certificate chain"
```
I've tried a few different options, such as setting the config to:
```
databaseDriverOptions: process.env.NODE_ENV !== "development" ?
      { ssl: { rejectUnauthorized: false } } : {}
  }
```
Added to the environment variable for DATABASE_URL to append ?sslmode=disable

I tried downloading the CA certificate and adding it to the repository and adding ?sslmode=require&sslrootcert=/ca-certificate.crt but this resulted in:
no such file or directory, open '/ca-certificate.crt'

I'm not sure where to go from here. Any ideas?

**Comment by nicolasdabreo** [comment]
I've also been struggling all day with this issue hosting with AWS

**Reply by vetaliieng** [reply]
Did you deploy it? What was the solution?

**Comment by vetaliieng** [comment]
Did you manage to deploy on DigitalOcean?

**Comment by Kyle-Lutz** [comment]
Im using app platform. You need to set sslmode to "no-verify"
